CHANGELOG.md

```diff
@@ -17,6 +17,13 @@ This changelog documents all notable user-facing changes of VAST.
   the query language entirely.
   [#1399](https://github.com/tenzir/vast/pull/1399)
 
+### ⚠️ Changes
+
+- VAST now ships with schema record types for Suricata's `mqtt` and `anomaly`
+  event types.
+  [#1408](https://github.com/tenzir/vast/pull/1408)
+  [@satta](https://github.com/satta)
```

Comment:
```suggestion
- ⚠️ VAST now ships with schema record types for Suricata's `mqtt` and `anomaly`
  event types.
  [#1408](https://github.com/tenzir/vast/pull/1408)
  [@satta](https://github.com/satta)
```

Please apply this diff and move to the top; we keep the changelog reverse-chronological in the Unreleased section instead of sorting it by category.